codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,14 +175,10 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
-        try {
-            FlowState state = state(stream);
-            state.newFrame(payload);
-            state.writeBytes(state.writableWindow());
-            flush();
-        } catch (Throwable e) {
-            payload.error(e);
-        }
+        FlowState state = state(stream);
+        state.newFrame(payload);
+        state.writeBytes(state.writableWindow());
+        flush();
```

Comment:
@Scottmitch I would just say if `stream(streamId)` throws an Exception, then our "contract" doesn't hold :-). There are multiple places where `writeBytes` gets called and none of them is wrapped in try catch. Also writeBytes might only throw an Exception if it's thrown by writeComplete or error. so we basically would have to put everything under try catch except for the writesBytes call, because we must not call error  after writeComplete or error has been called.